src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -292,6 +294,17 @@ private Task WriteFormattedInt32Async(int value, CancellationToken cancellationT
                     _connectionClose = true;
                 }
 
+                // Before creating the response stream, check to see if we're done sending any content,
+                // and propagate any exceptions that may have occurred.  The most common case is that
+                // the server won't send back response content until it's received the whole request,
+                // so the majority of the time this task will be complete.
+                Task sendRequestContentTask = _sendRequestContentTask;
```

Comment:
I don't understand this comment.  What happens if I do a POST with a large request body?  The socket Send will pend and the sendRequestContentTask will not be complete, correct?